src/test/spec/unified_runner/mod.rs

```diff
@@ -101,30 +101,74 @@ impl IntoFuture for RunUnifiedTestsAction {
     }
 }
 
-#[cfg_attr(feature = "tokio-runtime", tokio::test(flavor = "multi_thread"))]
-#[cfg_attr(feature = "async-std-runtime", async_std::test)]
-async fn test_examples() {
-    let _guard: RwLockWriteGuard<_> = LOCK.run_exclusively().await;
-    run_unified_tests(&["unified-test-format", "examples"]).await;
-}
-
 #[cfg_attr(feature = "tokio-runtime", tokio::test(flavor = "multi_thread"))]
 #[cfg_attr(feature = "async-std-runtime", async_std::test)]
 async fn valid_pass() {
     let _guard: RwLockWriteGuard<_> = LOCK.run_exclusively().await;
-    run_unified_tests(&["unified-test-format", "valid-pass"]).await;
+    #[cfg(feature = "in-use-encryption-unstable")]
+    let skipped_files = &[
+        // TODO RUST-1570: unskip this test (ditto below)
+        "collectionData-createOptions.json",
+        // TODO RUST-1405: unskip this test (ditto below)
+        "expectedError-errorResponse.json",
+        // TODO RUST-582: unskip these tests (ditto below)
+        "entity-cursor-iterateOnce.json",
+        "matches-lte-operator.json",
+        // TODO: unskip this test when the convenient transactions API tests are converted to the
+        // unified format (ditto below)
+        "poc-transactions-convenient-api.json",
+    ];
+    #[cfg(not(feature = "in-use-encryption-unstable"))]
+    let skipped_files = &[
+        "collectionData-createOptions.json",
+        "expectedError-errorResponse.json",
+        "entity-cursor-iterateOnce.json",
+        "matches-lte-operator.json",
+        "poc-transactions-convenient-api.json",
+        "kmsProviders-placeholder_kms_credentials.json",
+        "kmsProviders-unconfigured_kms.json",
+        "kmsProviders-explicit_kms_credentials.json",
+        "kmsProviders-mixed_kms_credential_fields.json",
+    ];
```

Comment:
couldn't come up with a more graceful way to do this without violating lifetime rules